public/components/report_definitions/report_settings/report_settings.tsx

```diff
@@ -162,22 +167,31 @@ export function ReportSettings() {
     target: { value: React.SetStateAction<string> };
   }) => {
     setReportName(e.target.value);
+    createReportDefinitionRequest['report_name'] = e.target.value.toString();
   };
 
   const handleReportDescription = (e: {
     target: { value: React.SetStateAction<string> };
   }) => {
     setReportDescription(e.target.value);
+    createReportDefinitionRequest['description'] = e.target.value.toString();
   };
 
   const handleReportSource = (e: React.SetStateAction<string>) => {
     setReportSourceId(e);
+    if (e === 'dashboardReportSource') {
+      createReportDefinitionRequest['report_source'] = 'Dashboard';
+    } else if (e === 'visualizationReportSource') {
+      createReportDefinitionRequest['report_source'] = 'Visualization';
+    }
   };
 
   const handleDashboardSelect = (e: {
     target: { value: React.SetStateAction<string> };
   }) => {
     setDashboardSourceSelect(e.target.value);
+    createReportDefinitionRequest['report_params']['url'] =
+      'http://localhost:5601/app/dashboards#/view/' + e.target.value;
```

Comment:
don't use hardcode localhost, could we use relative url?

Reply:
Will change